### ISSUE TITLE
Search box in the file import dialog

### ISSUE THREAD

**Issue by QueenHek** [hydrusnetwork/hydrus#300]
A simple search box added to the "add tags before import" submenu after drag n' dropping a folder into hydrus

![image](https://user-images.githubusercontent.com/46251909/85921437-a613ea80-b87c-11ea-9feb-c11207654b18.png)

a) "Search through the whole file path", so for example, inputting _samus_ in the box, would check if said word appears on a path, for example

C:\Hydrusimport\r34\femboy\samus

In this case, all files in said folder would appear and everything else would be excluded. Making it a quick way of mass tagging upon import

b) "Search through filename". In this case, it would be a file by file search in which Hydrus would try to match the sequence in the filename.

So for example, searching for _samus_ would land all files that have said word in their filename and everything else would be filtered.

Basically, this would be a basic RegEx filter for quick tagging for users that don't want to have to standarize filenames/paths and deal with regex.
